pkgs/development/libraries/gstreamer/validate/default.nix

```diff
@@ -3,7 +3,7 @@
 }:
 
 stdenv.mkDerivation rec {
-  name = "gst-validate-1.12.2";
+  name = "gst-validate-1.12.3";
```

Comment:
Forgot to update the hash (and test apparently).

Reply:
The same holds also for -vaapi, -python, -editing-services.

Reply:
Note: due to openssl security update, this now got to master, so any fixes should go there directly.

Reply:
Fixed in 9ebcc8db01cd1797a8bfcdd5b04643e6e92b6bb2